.github/workflows/ci.yml

```diff
@@ -36,26 +37,42 @@ jobs:
   #
   lint_github_actions:
     name: 'Lint (GitHub Actions)'
-    runs-on: 'ubuntu-latest'
+    runs-on: 'gemini-cli-ubuntu-16-core'
```

Comment:
I wonder if it's worthwhile for us to try to use this on `self-hosted` sooner than later.  As I understand, those runners do autoscaling.